sormas-backend/src/main/java/de/symeda/sormas/backend/common/AbstractCoreAdoService.java

```diff
@@ -35,4 +50,63 @@ public boolean isArchived(String uuid) {
 		long count = em.createQuery(cq).getSingleResult();
 		return count > 0;
 	}
+
+	protected <T extends ChangeDateBuilder<T>> T addChangeDates(T builder, From<?, ADO> adoPath, boolean includeExtendedChangeDateFilters){
+		return builder.add(adoPath);
+	}
+
+	public Date calculateCaseEndOfProcessingDate(List<String> entityuuids) {
```

Comment:
Is this method returning the maximum change date for a list of uuids of the same entity type?
At first glance this seems to be the wrong approach. The end of processing date needs to be calculated for every instance of the core entity type individually.

Reply:
update the logic, the end of processing date calculation is done separately on each instance